stellargraph/core/graph.py

```diff
@@ -211,27 +202,24 @@ def __init__(
                 ExperimentalWarning,
             )
 
-            self._graph = None
-
-            if nodes is None:
-                nodes = {}
-            if edges is None:
-                edges = {}
-
-            self._is_directed = is_directed
+        if nodes is None:
+            nodes = {}
+        if edges is None:
+            edges = {}
```

Comment:
Isn't this overwriting the result from the networkx conversion if there was a graph provided?

Reply:
`edges` won't be `None` in that case.

Reply:
Ah of course 😄 still asleep apparently